src/Page/Auth.elm

```diff
@@ -979,14 +1004,9 @@ viewSignupForm signupForm formErrors webData =
                 , id "termsAccepted"
                 , checked signupForm.termsAccepted
                 , onCheck <| \termsAccepted -> UpdateSignupForm { signupForm | termsAccepted = termsAccepted }
-                , required True
                 ]
                 []
-            , label [ class "form-check-label", for "termsAccepted" ]
-                [ text "Je m’engage à respecter les "
-                , a [ href Env.cguUrl, target "_blank" ] [ text "conditions d'utilisation" ]
-                , text ", et suis informé(e) que cette utilisation ne peut se faire que dans le cadre de la vente de produits sur le marché français."
-                ]
```

Comment:
Nit: you can use `Markdown.simple` and `String.replace "{url}" Env.cguUrl` to build this (it always enforce target _blank for links)